package.json

```diff
@@ -55,7 +55,9 @@
     "istanbul": "0.3.17",
     "jscs": "1.13.1",
     "jshint": "2.8.0",
-    "mocha": "2.2.5"
+    "mocha": "2.2.5",
+    "shebang-command": "1.0.0",
+    "shebang-regex": "2.0.0"
```

Comment:
There are `devDependencies`. They won't be installed during `npm install git-hooks`.

BTW, `git-hooks` cannot use dependencies because of [npm@2 bug](https://github.com/npm/npm/issues/8850).